### DISCUSSION TITLE
Website doesn't shows the UI but only text and links.

### DISCUSSION THREAD

**Discussion by barun4269** [firefly-iii/firefly-iii#9581] [Firefly III] [answered]
### Support guidelines

- [x] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [x] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/explanation/more-information/what-its-not/)
- [x] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been requested before.

### Description

Disclaimer: I have followed instructions on Docker installations method and I am not familiar with how docker works so what I got through here is only log in page have ui but without firefly-iii logo and the rest as shown in the image.![Image](https://github.com/user-attachments/assets/a88dabae-9e31-4bc5-8806-532e3939cd3c)

### Solution

As of now I get the site to works but only shown as in the picture and cleared all cache and cookies in my own browser but still same.

**Comment by JC5** [comment, marked as answer]
Hi, thanks for starting a discussion. 

This is usually caused by a missing TRUSTED_PROXIES header. Is it set? Is the value an asterisk?

**Reply by barun4269** [reply]
@JC5 thank you for verify that, that solved the issues. I just rebuild and redo with TRUSTED_PROXIES value an asterisk.
